tests/debugpy/test_step.py

```diff
@@ -56,7 +56,6 @@ def func():
         inner2_target = targets[0]["id"]
 
         session.request("goto", {"threadId": stop.thread_id, "targetId": inner2_target})
-        session.wait_for_next_event("continued")
```

Comment:
Not sure why pydevd does a continue, but the spec says once a goto request happens, just a stop event should fire:

https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Goto

Reply:
It's possible that pydevd implemented it like that when the spec didn't have that bit; it's pretty common for them to add clarifications like that later on if someone comes and asks how exactly it should behave.